MapboxSearch/sdk/src/main/java/com/mapbox/search/SearchEngine.kt

```diff
@@ -206,6 +205,42 @@ public interface SearchEngine {
         callback = callback,
     )
 
+    /**
+     * Function to retrieve the details for a given mapboxId. The callback will be invoked with
+     * a [SearchResult] on successful execution.
+     *
+     * Note that this method is only supported for a SearchEngine with [ApiType.SBS].
+     *
+     * @param mapboxId for the item to retrieve details for
+     * @param executor [Executor] used for events dispatching, default is the main thread
+     * @param callback used to receive the [SearchResult] on successful execution
+     * @return [AsyncOperationTask] object representing pending completion of the request
+     */
+    public fun retrieve(
+        mapboxId: String,
+        executor: Executor,
+        callback: SearchResultCallback,
+    ): AsyncOperationTask
+
+    /**
+     * Function to retrieve the details for a given mapboxId that dispatches events using the
```

Comment:
I think it's worth mentioning in documentation that this function works only for SBS api type and throws `UnsupportedOperationException` for others.

Reply:
A few lines down, I do mention it, but best to call out the exception explicitly.

Reply:
resolved in commit 67f549d20089dda723347a1f64136c12620b433a